Makefile

```diff
@@ -42,6 +42,8 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with
 	pip-compile --upgrade -o requirements/pip-tools.txt requirements/pip-tools.in
 	pip install -qr requirements/pip.txt
 	pip install -qr requirements/pip-tools.txt
+	sed 's/Django<4.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
These lines are not needed since the global constraint has already been changed to `Django<5.0` instead so these lines won't make any change in the file.

Reply:
@awais786 do you want to drop this now that the global common constraints are getting pulled in?